front/src/common/BootstrapSNCF/MultiSelectSNCF.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+
+type MultiSelectSNCFProps = {
+  multiSelectTitle: string;
+  multiSelectSubTitle?: string;
+  selectOptions: { label: string }[];
+  onChange: React.Dispatch<React.SetStateAction<string[]>>;
+  selectedValues: string[];
+  isSelectAll?: boolean;
+};
+
+const MultiSelectSNCF = ({
+  multiSelectTitle,
+  multiSelectSubTitle,
+  selectOptions,
+  onChange,
+  selectedValues,
+  isSelectAll,
```

Comment:
```suggestion
  allowSelectAll,
```
From what I understand, I think this name would be more appropriate

Reply:
Done